RocketCast/PodcastController.swift

```diff
@@ -15,7 +15,9 @@ class PodcastController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         setupView()
-        ModelBridge.sharedInstance.downloadPodcastXML("http://billburr.libsyn.com/rss") { (downloadedPodcast) in
+        let url = "http://atp.fm/rss"
+        let data = NSData(contentsOfURL:      NSURL(string: url)!)
```

Comment:
I think this is temporary code until we get Milton and Mo's  XML download thing happening. Mark it so.